README.md

```diff
@@ -1,23 +1,36 @@
+| ----------------------- |:---------------------:|:------------------------:|
 | Windows                 | [![Windows Status](https://ci.appveyor.com/api/projects/status/github/analogdevicesinc/libiio?svg=true)](https://ci.appveyor.com/project/analogdevicesinc/libiio/branch/master) | [![Latest Windows build](https://raw.githubusercontent.com/wiki/analogdevicesinc/libiio/img/win_box.png)](https://ci.appveyor.com/api/projects/analogdevicesinc/libiio/artifacts/libiio-setup.exe?branch=master)
-| OS X                    | [![OSX Status](https://api.travis-ci.org/analogdevicesinc/libiio.svg?branch=master&label=osx&passingTex=foo)](https://travis-ci.org/analogdevicesinc/libiio) |
-| Linux                   | [![Linux Status](https://api.travis-ci.org/analogdevicesinc/libiio.svg?branch=master&label=linux)](https://travis-ci.org/analogdevicesinc/libiio) |  [![Debian](https://raw.githubusercontent.com/wiki/analogdevicesinc/libiio/img/deb_box.svg)](http://swdownloads.analog.com/cse/travis_builds/latest_libiio.deb) [![RPM File](https://raw.githubusercontent.com/wiki/analogdevicesinc/libiio/img/rpm_box.png)](http://swdownloads.analog.com/cse/travis_builds/latest_libiio.rpm)|
+| OS X                    | [![OSX Status](https://api.travis-ci.org/analogdevicesinc/libiio.svg?branch=master&label=osx&passingTex=foo)](https://travis-ci.org/analogdevicesinc/libiio) | [![OS-X package](https://raw.githubusercontent.com/wiki/analogdevicesinc/libiio/img/osx_box.png)](http://swdownloads.analog.com/cse/travis_builds/latest_libiio.pkg)
+| Linux                   | [![Linux Status](https://api.travis-ci.org/analogdevicesinc/libiio.svg?branch=master&label=linux)](https://travis-ci.org/analogdevicesinc/libiio) |  [![Debian](https://raw.githubusercontent.com/wiki/analogdevicesinc/libiio/img/deb.png)](http://swdownloads.analog.com/cse/travis_builds/latest_libiio.deb) [![RPM File](https://raw.githubusercontent.com/wiki/analogdevicesinc/libiio/img/rpm.png)](http://swdownloads.analog.com/cse/travis_builds/latest_libiio.rpm)|
 
+If you use it, and like it - please let us know. If you use it, and hate it - please let us know that too. The goal of the project is to try to make Linux IIO devices easier to use on a variety of platforms. If we aren't doing that - we will try to make it better.
 
-Library for interfacing with IIO devices
+Feedback is appreciated (in order of preference):
 
-Weblinks:
+  * [Github trackers](https://github.com/analogdevicesinc/libiio/issues)
+  * [Analog Devices web forums](https://ez.analog.com/community/linux-device-drivers/linux-software-drivers)
+  * [The IIO mailing list](http://vger.kernel.org/vger-lists.html#linux-iio)
 
-http://wiki.analog.com/resources/tools-software/linux-software/libiio
 
-http://wiki.analog.com/resources/tools-software/linux-software/libiio_internals
+Weblinks:
+  * About IIO: https://wiki.analog.com/software/linux/docs/iio/iio
+  * Doxygen: http://analogdevicesinc.github.io/libiio/
```

Comment:
Maybe 'Documentation' instead of 'Doxygen', people may not know what Doxygen is.